app/app.component.ts

```diff
@@ -5,7 +5,6 @@ import { Observable } from "rxjs";
 
 import { ActivatedRoute } from "@angular/router";
 import { IpcService, Workspace, WorkspaceService } from "@batch-flask/ui";
-// import { MonacoLoader } from "@batch-flask/ui/editor";
 import { PermissionService } from "@batch-flask/ui/permission";
```

Comment:
still don't know how this made its way in. i think you should revert this change.